apps/studio/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "generate": "prisma generate",
+    "generate": "prisma generate && npm run lint:fix && npm run format:fix",
```

Comment:
nit: would prefer these to be in 2 separate commands (ie, we keep the current `generate` and have 1 extra command called `format:fix` or smth) but more a style preference than anything